packages/components/psammead-most-read/README.md

```diff
@@ -0,0 +1,62 @@
+<!-- prettier-ignore -->
+# psammead-most-read - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-most-read%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-most-read%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-most-read)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-most-read) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-most-read)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-most-read&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/most-read--default-ltr) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-most-read.svg)](https://www.npmjs.com/package/@bbc/psammead-most-read) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `MostRead` component is designed to be used on the `article` pages, which are pages that display articles/stories. It's designed to have a numerical count and a link to another article.
+
+## Installation
+
+`npm install @bbc/psammead-most-read`
+
+## Props
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| item | object | yes | N/A | `{ header: 'This is a item', href: 'https://www.bbc.com' }` |
+| count | string | yes | N/A | `'rtl'`  |
+| script | object | yes | N/A | `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| service | string | yes | N/A | `'news'` |
+| dir | string | no | `'ltr'` | `'rtl'`  |
+
+## Usage
+
+A typical use-case of this component is as displayed below. It contains a count and an info elements. The info element is a link which point to the corresponding article while the count is a number that represent the position of the info element on the list of info elements.
+
+```jsx
+import React from 'react';
+import MostRead from '@bbc/psammead-most-read';
+
+const item = {
+    header: 'Cranberries singer O'Riordan died by drowning,
+    href: 'https://www.bbc.com'
+}
+
+<MostRead item={item} count="1" service="news" dir="ltr" />;
+
+```
+
+![most-read-single-item](documentation/most-read-single-item.png)
```

Comment:
I don't think we need to have a static image in our package, we can just use github's hosting: ![image](https://user-images.githubusercontent.com/30599794/67940838-59342300-fbcc-11e9-89e0-3982543596cf.png)

`![image](https://user-images.githubusercontent.com/30599794/67940838-59342300-fbcc-11e9-89e0-3982543596cf.png)`